services/core-web/common/components/tailings/TailingsSummaryPage.tsx

```diff
@@ -15,22 +14,11 @@ import {
   fetchTailingsStorageFacility,
   updateTailingsStorageFacility,
 } from "@mds/common/redux/actionCreators/mineActionCreator";
-import { flattenObject } from "@common/utils/helpers";
-import {
-  FormErrors,
-  getFormSyncErrors,
-  getFormValues,
-  InjectedFormProps,
-  isDirty,
-  reduxForm,
-  submit,
-  touch,
-} from "redux-form";
-
+import { isDirty } from "redux-form";
 import ArrowLeftOutlined from "@ant-design/icons/ArrowLeftOutlined";
 import BasicInformation from "@mds/common/components/tailings/BasicInformation";
-import Step from "@common/components/Step";
-import SteppedForm from "@common/components/SteppedForm";
+import Step from "@mds/common/components/forms/Step";
+import SteppedForm from "@mds/common/components/forms/SteppedForm";
```

Comment:
Got rid of SteppedForm in core/MS in exchange for common file! Worked shockingly well.